source/_integrations/samsungtv.markdown

```diff
@@ -90,12 +90,9 @@ media_content_type: channel
 
 It's possible to switch between the 2 sources `TV` and `HDMI`.
 
-### Home Assistant Core additional requirements
+### Known issues and restrictions
 
-You will need to install the `websocket-client` Python package in your Home Assistant install. This will probably be done with:
+#### Subnet/VLAN
 
-```bash
-pip3 install websocket-client
-```
-
-Remembering to activate your venv if you're using a venv install.
+Samsung SmartTV do not allow websocket connections across different subnets or VLANs. If your TV is not on the same subnet as Home Assistant this will fail.
```

Comment:
```suggestion
Samsung SmartTV does not allow WebSocket connections across different subnets or VLANs. If your TV is not on the same subnet as Home Assistant this will fail.
```